src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -389,12 +394,14 @@ private void DeleteValue(TransactionOperationContext context, string type, Blitt
                 if (delCmd.Name.StartsWith("db/"))
```

Comment:
What isn't this using a constant? And it probably need to use case insensitive match.